iib/workers/tasks/build.py

```diff
@@ -1498,7 +1498,17 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     """
     package_name, labels = _apply_package_name_suffix(metadata_path, organization)
 
-    operator_manifest = OperatorManifest.from_directory(manifests_path)
+    try:
+        operator_manifest = OperatorManifest.from_directory(manifests_path)
+    except ruamel.yaml.YAMLError as e:
+        error = f'Not a valid yaml file: {e}'
+        log.exception(error)
+        raise IIBError(error)
+    except (RuntimeError, ValueError) as e:
```

Comment:
@lkolacek, I came back from PTO and forgot to refresh the tab. I did a silly thing an commented on an outdated diff. Could you please have a look at the second part of my [comment](https://github.com/release-engineering/iib/pull/209#discussion_r570972525) where I suggest logging from the error callback? 

We might not need to catch `RuntimeError` and `ValueError` here.

Reply:
I added `@request_logger` as you requested. 

Catching also other exceptions except `ruamel.yaml.YAMLError` suggested Martin and I agree with him. Even though there is only a slight chance that this problem will ever occur.